packages/flutter/lib/src/rendering/selection.dart

```diff
@@ -627,6 +628,9 @@ class SelectionGeometry {
   /// The status of ongoing selection in the [Selectable] or [SelectionHandler].
   final SelectionStatus status;
 
+  /// The rects that represent the selection if there is any.
+  final List<Rect>? selectionRects;
```

Comment:
what coordinates are these rects in?

Reply:
These should be in the local coordinates of the Selectable. I'll update the docs.

Reply:
you can give it a default value of `const <Rect>[]` and make it not nullable.